src/js/components/Footer.jsx

```diff
@@ -0,0 +1,58 @@
+import {grey600} from 'material-ui/styles/colors'
+import {footerLeftText, footerRightText} from '../../../config'
+import React, {Component} from 'react'
+
+const style = {
+  bottomLeft: {
+    display: 'inline-block'
+  },
+  bottomRight: {
+    float: 'right'
+  },
+  footer: {
+    alignItems: 'center',
+    background: grey600,
+    color: 'white',
+    display: 'flex',
+    fontSize: '0.8rem',
+    padding: '2px 5px'
+  },
+  spacer: {
+    flex: 1
+  }
+}
+
+const normalizeFooterText = (arg) => {
+  if (Array.isArray(arg)) {
+    return arg
+  }
+  if (typeof arg === 'string') {
+    return [arg]
+  }
+  
+  return ''
```

Comment:
I think the problem I am pointing out is that this function does three things

1. If its array, just return the passed in arg
2. If its a string, return an array where the arg is the only item in the array
3. If netiher 1 nor 2 are true, than return a string

number 3 is confusing. it should return an empty array if you are "normalizing" since all other output is an array